lib/dotcom/realtime_schedule.ex

```diff
@@ -17,16 +17,17 @@ defmodule Dotcom.RealtimeSchedule do
   alias Routes.Route
   alias Schedules.RepoCondensed, as: SchedulesRepo
   alias Schedules.ScheduleCondensed
-  alias Stops.Repo, as: StopsRepo
   alias Stops.Stop
 
+  @stops_repo Application.compile_env!(:dotcom, :repo_modules)[:stops]
+
   # the long timeout is to address a worst-case scenario of cold schedule cache
   @long_timeout 15_000
 
   @predicted_schedules_per_stop 2
 
   @default_opts [
-    stops_fn: &StopsRepo.get/1,
+    stops_fn: Function.capture(@stops_repo, :get, 1),
```

Comment:
We can remove the function injection opts, just call `@stops_repo.get` where needed